mutiny-wasm/src/indexed_db.rs

```diff
@@ -270,6 +268,20 @@ impl IndexedDbStorage {
                     None => return Ok(Some((kv.key, obj.value))),
                 }
             }
+            DEVICE_LOCK_KEY => {
+                let obj = vss.get_object(&kv.key).await?;
+                // we can get version from device lock, so we should compare
+                match current.get_data::<DeviceLock>(&kv.key)? {
+                    Some(versioned) => {
+                        if versioned.time <= kv.version
```

Comment:
Comparing time with version?

Reply:
we use the time as the vss version number so it is always increasing

Reply:
Ah interesting. There's no number conversion issues then right? I remember we had to do some number conversions before.

Reply:
yeah both u32